timeout_config.hh

```diff
@@ -19,11 +23,46 @@ struct timeout_config {
     db::timeout_clock::duration truncate_timeout;
     db::timeout_clock::duration cas_timeout;
     db::timeout_clock::duration other_timeout;
```

Comment:
Isn't it fragile to leave these numbers as-is? E.g. previous version lost alternator copy of some of those values. Why not patch _all_ the above fields to be `updateable_value<duration>`s and patch the users/callers/passers/etc. accordingly? The updateable_value can be easily subscribed on db::config::named_value change

Reply:
Yes. Look at, e.g., `struct restrictions_config`. It has two "integers" onboard, booth are live-updateable via db::config. It's exactly what this PR is trying to achieve with timeout_config and my point is that converting all its inhabitants to be `updateable_value<duration>` is the safest way to go

Reply:
i see. thanks! makes sense to me.